sys/net/include/etx_beaconing.h

```diff
@@ -52,15 +55,15 @@ extern "C" {
  * and 5 (For ETX_MAX_JITTER) unless those values are adjusted too.
  */
 #define ETX_INTERVAL        (1000)
-#define ETX_WINDOW          (10)                    //10 is the default value
-#define ETX_BEST_CANDIDATES (15)                    //Sent only 15 candidates in a beaconing packet
-#define ETX_TUPLE_SIZE      (2)                     //1 Byte for Addr, 1 Byte for packets rec.
-#define ETX_PKT_REC_OFFSET  (ETX_TUPLE_SIZE - 1)    //Offset in a tuple of (addr,pkt_rec), will always be the last byte
-#define ETX_IPV6_LAST_BYTE  (15)                    //The last byte for an ipv6 address
-#define ETX_MAX_JITTER      (ETX_INTERVAL / 5)      //The default value is 20% of ETX_INTERVAL
-#define ETX_JITTER_MOD      (ETX_MAX_JITTER + 1)    //The modulo value for jitter computation
-#define ETX_DEF_JIT_CORRECT (ETX_MAX_JITTER / 2)    //Default Jitter correction value (normally ETX_MAX_JITTER / 2)
-#define ETX_CLOCK_ADJUST    (52500)                 //Adjustment for clockthread computations to stay close/near ETX_INTERVAL
+#define ETX_WINDOW          (10)                    /** 10 is the default value */
```

Comment:
Wrong opening tag, use `/**< blah blah */` to document something that comes before the documentation, or move the documentation to the line above the definition.